sdks/python/apache_beam/ml/inference/base_test.py

```diff
@@ -79,12 +79,20 @@ def batch_elements_kwargs(self):
     return {'min_batch_size': 9999}
 
 
-class FakeModelHandlerExtraInferenceArgs(FakeModelHandler):
+class FakeModelHandlerFailsOnInferenceArgs(FakeModelHandler):
+  def run_inference(self, batch, unused_model, inference_args=None):
+    raise ValueError('run_inference should not be called.')
```

Comment:
```suggestion
    raise ValueError('run_inference should not be called because error should already be thrown from the validate_inference_args check.')
```